File: Atividade de revisão/Ex_4.c
#include <stdio.h>
int main() {
  int Matriz[10][10];            
  int i, j, Maior_Elemento, Linha_Do_Maior, Coluna_Do_Maior;      
  for(i = 0; i < 10; i++){       
    for(j = 0; j < 10; j++){
      printf("Digite o elemento da posicao[%d][%d] : ", (i+1), (j+1));
      scanf("%d", &Matriz[i][j]);
    }
  }
  Maior_Elemento = Matriz[0][0];         
  for(i = 0; i < 10; i++){                
    for(j = 0; j < 10; j++){              
      if (Matriz[i][j] > Maior_Elemento){
        Maior_Elemento = Matriz[i][j];
        Linha_Do_Maior = i;
        Coluna_Do_Maior = j;
      }  
    }
  }
  printf("\n");
  printf("Maior valor encontrado: %d \n", Maior_Elemento);
  printf("Elementos da coluna: ");
  for(i = 0; i < 10; i++){
    printf("%d ", Matriz[i][Coluna_Do_Maior]);
  }
  printf("\n");
  printf("Elementos da linha: ");
  for(i = 0; i < 10; i++){
    printf("%d ", Matriz[Linha_Do_Maior][i]);
  }
  return 0;
}

File: Estrutura De Dados - C++/Atividade 1 - Revisão/Ex_4.py
'''
Elabore um programa que leia uma matriz de dimensões informadas pelo usuário. 
Localize o maior elemento da matriz e informe sua posição.
'''

def Cria_Matriz(Matriz,Linhas,Colunas):
    for i in range(Linhas):
        Matriz.append([0] * Colunas)

def Insere_Elementos_Na_Matriz(Matriz,Linhas,Colunas):
    for i in range(Linhas):
        for j in range(Colunas):
            Matriz[i][j] = int(input("Qual o elemento da matriz [" + str(i+1) + "][" + str(j+1) + "] ? ")) 

def Imprime_Matriz (Matriz,Linhas,Colunas):
    for i in range(Linhas):
        for j in range(Colunas):
            print(Matriz[i][j], end= "   ")
        print()
        print()

def Encontra_Maior(Matriz,Linhas,Colunas):
    Maior = Matriz[0][0]
    Linha_Maior = 0
    Coluna_Maior = 0
    for i in range(Linhas):
        for j in range(Colunas):
            if Matriz[i][j] > Maior :
                Maior = Matriz[i][j]
                Linha_Maior = i
                Coluna_Maior = j
    return Maior, Linha_Maior, Coluna_Maior
Matriz = []
print()
Linhas = int(input("Digite a quantidade de linhas da matriz : "))
print()
Colunas = int(input("Digite a quantidade de colunas da matriz : "))
print()
Cria_Matriz(Matriz, Linhas, Colunas)
Insere_Elementos_Na_Matriz(Matriz, Linhas, Colunas)
Maior, Linha_Maior, Coluna_Maior = Encontra_Maior(Matriz, Linhas, Colunas)
print()
Imprime_Matriz(Matriz, Linhas, Colunas)
print()
print("O maior elemento da matriz é o numero", Maior , "e ele se encontra na Linha", (Linha_Maior + 1), "Coluna" , (Coluna_Maior + 1))
